--- localization.py	2025-02-06 15:14:15.675457542 +0100
+++ localization-b.py	2025-02-06 15:11:09.340186764 +0100
@@ -16,6 +16,7 @@
 
 # Set application version, and icon
 v = "3.5.2"
+v += "-beta"
 icon = "io.github.vikdevelop.SaveDesktop"
 rel_notes = "<p>Fixed a bug with loading the password requirement in the Cinnamon DE for unlocking the Nextcloud folder, and added a notification about starting synchronization.</p>\
 <p>3.5.1</p>\
